src/LanguageServer/Impl/Implementation/Server.cs

```diff
@@ -99,6 +99,7 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
             _editorFiles = new EditorFiles(this);
 
             SymbolIndex = new SymbolIndex();
+            RunningDocumentIndexer = new RunningDocumentIndexer(SymbolIndex, PythonLanguageVersion.V37);
```

Comment:
Service container has `IPythonInterpreter` that can tell you the language version.